### DISCUSSION TITLE
StackOverflow -> StackExchange

### DISCUSSION THREAD

**Discussion by jayvdb** [lowlighter/metrics#641] [Ideas]
It would be useful for the StackOverflow plugin to be generalised a bit to be a StackExchange plugin.

For some people, the SuperUser, Android, U&L, etc may be where they are more active/more suitable for their GitHub profile.

**Comment by lowlighter** [comment]
Yeah I guess if the endpoints are similar it could be done, but I didn't dig too much into it to be honest.

I'm not an expert with stack exchange APIs but eventually if there's a lot of demands for widening the plugin (or if someone want to give it a shot) I'll look into it 🙂
